apps/passport-server/src/services/multiProcessService.ts

```diff
@@ -24,6 +24,53 @@ const WORKER_MODULE_PATH = path.join(
 
 const DEFAULT_WORKER_QUANTITY = os.cpus().length;
 
+/**
+ * Very simple in-memory Least-Recently-Used cache
+ */
+class LruCache<T> {
```

Comment:
may be nice to use an existing implementation, like https://www.npmjs.com/package/lru-cache, otherwise we're introducing more surface area for bugs.

if you're sure you don't have bugs, feel free to just use this.

Reply:
I'll change it to use an existing implementation. It's only like this because I wrote it on the plane and didn't have access to npm.

Reply:
10x gigbrain move ^

Reply:
> I'll change it to use an existing implementation. It's only like this because I wrote it on the plane and didn't have access to npm.

10x gigabrain move ^